### PR TITLE
React to changes in razor tooling

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by javiercn** [comment]
@NTaylorMullen

File: samples/MvcSandbox/project.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-*",
     "Microsoft.AspNetCore.Diagnostics": "1.0.0-*",
-    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc": "1.0.0-*",
-    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.0.0-*",
+    "Microsoft.AspNetCore.Razor.Tools": "1.0.0-*",
```
Comment (NTaylorMullen) [inline samples/MvcSandbox/project.json]: build time only package reference in dependenceis

**Comment by NTaylorMullen** [comment]
:shipit: once you change the dependencies section reference to be build only.

File: samples/MvcSandbox/project.json (hunk 1 above)
Comment (javiercn) [inline samples/MvcSandbox/project.json]: Do we need to do this for the IIS publish tool too?

Comment (NTaylorMullen) [inline samples/MvcSandbox/project.json]: @moozzyk could answer that.

**Pull request merged**